service/mapper/transitions.go

```diff
@@ -58,19 +59,34 @@ func NewTransitions(log zerolog.Logger, load Loader, chain dps.Chain, feed Feede
 		load:  load,
 		chain: chain,
 		feed:  feed,
-		index: index,
+		read:  read,
+		write: write,
 		once:  &sync.Once{},
 	}
 
 	return &t
 }
 
-// BootstrapState bootstraps the state by loading the checkpoint if there is one and initializing
-// the elements subsequently used by the FSM.
-func (t *Transitions) BootstrapState(s *State) error {
+// InitializeMapper will initialize the mapper by either going into
+// bootstrapping or into resuming, depending on the configuration.
```

Comment:
```suggestion
// InitializeMapper initializes the mapper by either going into
// bootstrapping or into resuming, depending on the configuration.
```

Since this is not a code comment but something that goes into the godoc, IMO future tense has no place here